### PR TITLE
 Shortcut to open help menu - Shift + F1

### PR DESCRIPTION
**Pull request by vorabrijesh**
This mentions issues 

#24211    #3786

I have tried to add the shortcut to open help menu: Shift + F1 which is the standard way of opening help menu and since many of the people needed it.

### CODE REVIEW

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -5540,7 +5540,7 @@ EditorNode::EditorNode() {
 	p = help_menu->get_popup();
 	p->set_hide_on_window_lose_focus(true);
 	p->connect("id_pressed", this, "_menu_option");
-	p->add_icon_shortcut(gui_base->get_icon("HelpSearch", "EditorIcons"), ED_SHORTCUT("editor/editor_help", TTR("Search"), KEY_F4), HELP_SEARCH);
+    p->add_icon_shortcut(gui_base->get_icon("HelpSearch", "EditorIcons"), ED_SHORTCUT("editor/editor_help", TTR("Search"), KEY_MASK_SHIFT | KEY_F1), HELP_SEARCH);
```
Comment (bojidar-bg) [inline editor/editor_node.cpp]: Wrong indentation: should be one tab, but is four spaces.

Comment (vorabrijesh) [reply editor/editor_node.cpp]: I resolved the issue can you please review again if now the issue of help shortcut is resolved or not.

File: editor/editor_node.cpp [hunk 2]
```diff
@@ -5540,7 +5540,7 @@ EditorNode::EditorNode() {
 	p = help_menu->get_popup();
 	p->set_hide_on_window_lose_focus(true);
 	p->connect("id_pressed", this, "_menu_option");
-	p->add_icon_shortcut(gui_base->get_icon("HelpSearch", "EditorIcons"), ED_SHORTCUT("editor/editor_help", TTR("Search"), KEY_F4), HELP_SEARCH);
+    	p->add_icon_shortcut(gui_base->get_icon("HelpSearch", "EditorIcons"), ED_SHORTCUT("editor/editor_help", TTR("Search"), KEY_MASK_SHIFT | KEY_F1), HELP_SEARCH);
```
Comment (bojidar-bg) [inline editor/editor_node.cpp]: Now you have 4 spaces and a tab. Getting there!

Comment (vorabrijesh) [reply editor/editor_node.cpp]: sorry for my mistake

**Comment by bojidar-bg** [comment]
One final thing: would be nice if you rebase your branch so you have only one commit in the PR. Check [the docs](http://docs.godotengine.org/en/3.1/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase) for more info on how to do it.

**Comment by vorabrijesh** [comment]
I tried to do that but now when I compare with my master and my updated branch I get 51 commits instead of just my commit.
So, how do I delete other commits so that just my commit remains

**Comment by vorabrijesh** [comment]
I closed this PR and opened a new PR which had just one commit  #27151,
please review that PR

**Pull request closed without merging**